File: Algoritmo/ProjetoPython/ex032.py
# Escreva um programa que leia um caractere diga se ele é uma vogal, consoante, número ou um símbolo (qualquer
# outro caracter, que não uma letra ou número).

print('*'*85)
print('Esse programa diz se o valor digitado é uma vogal, consoante, número ou um símbolo')
print('*'*85)
caracter = input('Digite aqui: ').upper()[0]

if caracter in 'AEIOU':
    print('É uma vogal')
elif caracter in 'BCDFGJKLMNPQRSTVWXZ':
    print('É uma consoante')
elif caracter in '12345678910':
    print('É um número')
elif caracter in '!@#$%¨&*()_+°/?=§´~^\|;,.':
    print('É um símbolo')

File: Algoritmo/1-SEMESTRE/ProjetoJava/src/ExerciciosPDF/ex032.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ExerciciosPDF;

import javax.swing.JOptionPane;

/**
 *
 * @author vitor
 */
public class ex032 {
    public static void main(String[] args) {
        JOptionPane.showMessageDialog(null, "Esse programa diz se o valor digitado é uma vogal, consoante, número ou um símbolo");
        
        String caracter;
        caracter = JOptionPane.showInputDialog("Digite aqui: ");
        
        
        String[] vogal = {"A","E","I","O","U"};
        String[] consoante = {"B","C","D","F","G","J","K","L","M","N","P","Q","R","S","T","V","W","X","Z"};
        int[] numero = {1,2,3,4,5,6,7,8,9,10};
        
        int i;
        
        for (i=0; i <5; i++){
            String toUpperCaracter = caracter.toUpperCase();
            if ((vogal[i].equals(toUpperCaracter))){
                System.out.println("É uma vogal");
            }
        }
        
        for (i=0; i <19; i++){
            String toUpperCaracter = caracter.toUpperCase();
            if ((consoante[i].equals(toUpperCaracter))){
                System.out.println("É uma consoante");
            }
        }
        
        for (i=0; i <numero.length; i++){
            int num = Integer.parseInt(caracter);
            if (numero[i] == num){
                System.out.println("É um numero");
            } 
        }       
    }
}
